frontend/src/app/main/ui/workspace/header.cljs

```diff
@@ -464,7 +468,8 @@
                 :project project
                 :file file
                 :team-id team-id
-                :page-id page-id}]]
+                :page-id page-id
+                :workspace-read-only? workspace-read-only?}]]
```

Comment:
You don't need to pass this all down having the react context set. Just use the use-ctx on the component if you need to know the value

Reply:
Fixed!